Formula/sendmidi.rb

```diff
@@ -0,0 +1,17 @@
+class Sendmidi < Formula
+  desc "Multi-platform command-line tool to send out MIDI messages"
+  homepage "https://github.com/gbevin/SendMIDI"
+  url "https://github.com/gbevin/SendMIDI/archive/1.0.3.tar.gz"
+  sha256 "f37a4ff16fac73e23e403fee0ed7ffb0eefcca399ed39dece9cd157969ad5fc6"
+
```

Comment:
Thanks! Sadly I don't really see how to do anything more since it relies on MIDI devices being connected to the computer. Without a MIDI device, the tool does nothing that's useful.